cql3/statements/create_table_statement.cc

```diff
@@ -452,6 +453,20 @@ std::optional<sstring> check_restricted_table_properties(
             break;
         }
     }
+    if (strategy && *strategy == sstables::compaction_strategy_type::time_window){
+        std::map<sstring, sstring> options = cfprops.get_compaction_type_options();
+        sstables::time_window_compaction_strategy_options _twcs_options(options);
+        auto ttl = cfprops.get_default_time_to_live();
+        int max_windows = qp.db().get_config().twcs_max_window_count();
+
+        if (ttl > 0){
+           int window_count = (int) ttl / _twcs_options.get_sstable_window_size().count();
```

Comment:
We avoid C-style cases, they are dangerous. In this case, why cast at all? It just provides opportunity for overflow.

Time is best divided as two chrono::duration objects so we don't lose track of the units.